src/data-workspace/inputs/date-input.js

```diff
@@ -15,47 +14,49 @@ export const DateInput = ({
     cocId,
     deId,
     disabled,
-    fieldname,
-    form,
     locked,
     onFocus,
+    onBlur,
     onKeyDown,
+    initialValue,
+    setValueSynced,
 }) => {
+    const [value, setValue] = useState(initialValue)
+    const [lastSyncedValue, setLastSyncedValue] = useState(initialValue)
+    const [syncTouched, setSyncTouched] = useState(false)
+
+    useEffect(() => {
+        if (syncTouched) {
+            setValueSynced(value === lastSyncedValue)
+        }
+    }, [value, lastSyncedValue, syncTouched])
```

Comment:
If exhaustive deps?

Reply:
🤦‍♂️ sorry for that. I'm impressed(?) with myself for managing to miss multiple lint warnings both locally and in github 😬